packages/select/src/Select/components/Trigger/TriggerButtonBase/TriggerButtonBase.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { HTMLAttributes } from "react"
+import { useFocusRing } from "@react-aria/focus"
+import { mergeProps } from "@react-aria/utils"
+import classNames from "classnames"
+import { OverrideClassName } from "@kaizen/component-base"
+import { Icon } from "@kaizen/component-library"
+import chevronDown from "@kaizen/component-library/icons/chevron-down.icon.svg"
+import chevronUp from "@kaizen/component-library/icons/chevron-up.icon.svg"
+
+import { useMenuTriggerContext } from "../../../provider/MenuTriggerProvider"
+import styles from "./TriggerButtonBase.module.scss"
+
+export interface TriggerButtonBaseProps
+  extends OverrideClassName<HTMLAttributes<HTMLDivElement>> {
+  children: React.ReactNode
+}
+
+export const TriggerButtonBase: React.VFC<TriggerButtonBaseProps> = ({
```

Comment:
Question: Is this a Filter button? Or something else? 
Because there's one inside the `FilterMultiSelect`, so it's very confusing when they share the same name if they are not the same thing (I understand we are not applying DRY yet because our Filter pattern needs more thought).